src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/App.config

```diff
@@ -167,6 +167,10 @@
                 <endpoint address="tcp-nosecurity-callback" binding="netTcpBinding" bindingConfiguration="nettcpBindingCallback"
                           name="nettcp-defaultBinding" contract="WcfService.IDuplexChannelService" />
             </service>
+            <service name="WcfService.DuplexChannelCallbackReturnService">
+                <endpoint address="tcp-nosecurity-callback" binding="netTcpBinding" bindingConfiguration="nettcpBindingCallback"
+                          name="nettcp-defaultBinding" contract="WcfService.IWcfDuplexTaskReturnService" />
+            </service>
```

Comment:
Depending who merges first, @dmetzgar may need to update his bridge branch to accommodate thisnew endpoint.